tests/acceptance-bdd/literature_suggestion.feature

```diff
@@ -0,0 +1,306 @@
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I click on the button Skip, and fill the form manually
+    And I insert autrans in the autocomplete input box with id conf_name
+    Then I should see IN2P3 School of Statistics, 2012-05-28, Autrans, France in the input box with id conf_name
+
+
+  # Basic info autocomplete for affiliation
+  Scenario: Use the autocomplete for the affilitaion
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I click on the button Skip, and fill the form manually
+    And I insert oxf in the autocomplete input box with id authors-0-affiliation
+    Then I should see Oxford U. in the input box with id authors-0-affiliation
+
+
+  # Import an article with the arXiv id
+  Scenario: Import an article using the arXiv id
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I insert <input> in the input box with id arxiv_id
+    And I click on the button with id importData
+    And I click on the button with id acceptData
+    Then I should see International Journal of Theoretical Physics in the input box with id journal_title
+    And I should see 4 in the input box with id issue
+    And I should see 1999 in the input box with id year
+    And I should see 38 in the input box with id volume
+    And I should see 1113-1133 in the input box with id page_range_article_id
+    And I should see Maldacena, Juan in the input box with id authors-0-name
+    And I should see 10.1023/A:1026654312961 in the input box with id doi
+    And I should see The Large N Limit of Superconformal Field Theories and Supergravity in the input box with id title
+    And I should see We show that the large $N$ limit of certain conformal field theories in the input box with id abstract
+
+  Examples: Vertical
+    | input | hep-th/9711200 |
+
+
+  # Import an article with the DOI id
+  Scenario: Import an article using the DOI id
+    Given I am logged in
+    When I go to the literature suggestion page
+    And I insert <input> in the input box with id doi
+    And I click on the button with id importData
+    And I click on the button with id acceptData
+    Then I should see The Astrophysical Journal in the input box with id journal_title
+    And I should see 2 in the input box with id issue
+    And I should see 1998 in the input box with id year
+    And I should see 500 in the input box with id volume
+    And I should see 525-553 in the input box with id page_range_article_id
+    And I should see Schlegel, David J. in the input box with id authors-0-name
+    And I should see 10.1086/305772 in the input box with id doi
+    And I should see Maps of Dust Infrared Emission for Use in Estimation of Reddening and Cosmic Microwave Background Radiation Foregrounds in the input box with id title
+
+  Examples: Vertical
+    | input | 10.1086/305772 |
+
+
+  Scenario: Submit thesis and verify record detail in the list page
```

Comment:
Make it work
make it right 
make it realistic 
make it fast

I will add a test with unicode in this PR